includes/class-wi-volunteer-management.php

```diff
@@ -81,6 +81,7 @@ public function __construct() {
 
 		// Load the public hooks
 		$this->define_public_hooks();
+      $this->define_widget_hooks();
```

Comment:
@jg314 

Ah, I gotchya. My fault. I had already removed the define_widget_hooks function but I will go ahead and move the register_widget function back into widget/class-widget.php and reference that class when calling the add_action method for the widget within the define_public_hooks function. Sorry about that.